### ISSUE TITLE
search_issues() Broken for Python 3 in 0ad3c352839060da9dc146df635a97119eccd114 

### ISSUE THREAD

**Issue by milo-minderbinder** [pycontribs/jira#138]
In commit 0ad3c352839060da9dc146df635a97119eccd114 reference on line 1580 of jira.client module to `basestring` type, which is non-existent in Python 3, breaks the `search_issues()` method of the jira.client.JIRA class.

**Comment by muffl0n** [comment]
According to http://python-future.org/compatible_idioms.html#basestring it would be sufficient to just do this:
```
# Python 2 and 3: alternative 1
from past.builtins import basestring
```

**Comment by filmackay** [comment]
Fixed in outstanding PR #141.
